testssl.sh

```diff
@@ -1067,8 +1069,20 @@ count_ciphers() {
      echo $(wc -w <<< "${1//:/ }")
 }
 
+#arg1: TLS 1.2 and below ciphers
+#arg2: TLS 1.3 ciphers
+#arg3: options (e.g., -V)
 actually_supported_ciphers() {
```

Comment:
Actually your PR made clear somehow that $OPENSSL is called often (atm: 27x) to generate lists.

I was wondering whether it would make sense for longer terms to have a variable which contains everything  (``ALL:COMPLEMENTOFALL:@STRENGTH``) and to extract here the ones needed?